packages/office-ui-fabric-react/src/components/HoverCard/examples/HoverCard.Target.Example.tsx

```diff
@@ -84,11 +84,13 @@ export class HoverCardTargetExample extends React.Component<{}, {}> {
     return (
       <Fabric>
         <p>
-          Hover over the <i>key</i> cell of a row item to see the card or use the keyboard to navigate to it.
+          Hover over the <i>key</i> cell of a row item to see the card or use the keyboard to navigate to it by tabbing to a row and hitting
+          the right arrow key.
         </p>
         <p>
-          When using the keyboard to tab to it, open the card with the hotKey and it will automatically focus the first focusable element in
-          the card allowing further navigation inside the card.
+          When using the keyboard to navigate, open the card with the hotKey (as defined by the hotKey prop, which is defined as 'enter' in
+          the following example) and it will automatically focus the first focusable element in the card allowing further navigation inside
+          the card.
```

Comment:
Maybe?

```text
When using the keyboard to navigate, open the card while pressing the hotKey (as defined by the hotKey prop, which is the `Enter` key in the following example). This will automatically focus the first focusable element in the card, allowing further navigation inside of it.
```

Reply:
It is a bit runny. I'll just break out the parenthesized content as a followup sentence.